cedar-lean/DiffTest/Parser.lean

```diff
@@ -133,6 +133,12 @@ def jsonToExtFun (json : Lean.Json) : ParseResult ExtFun := do
   | "isLoopback" => .ok .isLoopback
   | "isMulticast" => .ok .isMulticast
   | "isInRange" => .ok .isInRange
+  | "datetime" => .ok .datetime
```

Comment:
The changes in this file conflict with https://github.com/cedar-policy/cedar-spec/pull/567, which deletes this file.  I guess it's a race to see which PR we merge first :) Either way, the correct end state is that this file is deleted

Reply:
True, I've removed the changes in that file to avoid conflicts.